DataCapturing/Model/PersistenceLayer.swift

```diff
@@ -314,17 +326,25 @@ public class PersistenceLayer {
      - toMeasurement: The measurement to save the provided ojects to.
     */
     func syncSave(locations: [GeoLocation], accelerations: [Acceleration], toMeasurement measurement: MeasurementEntity) {
+        // FIXME: this does not work. Most likely because the first save call changes the measurement in the database but does not refresh it as an object, thus the second call to save fails.
```

Comment:
Haben wir eine Github Procedure die sichergeht, dass wir keinen PR in den master releasen können, der FIXMEs enthält? Sonst wäre sowas hier gefährlich, könnte man übersehen beim Merge => master?